guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>We can use a combination of the <code>geocoder</code> and <code>out_sr</code> parameters to get the geocoded results using a custom geocoder and output in the desired spatial reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>